### PR TITLE
Fix many minor code style issues

### PR DESCRIPTION
**Pull request by thiemowmde**
I found that all these changes are to minor for individual patches. They also interfere with each other. If you disagree with a change but want to merge the rest, please tell me, and I will split it off.
- ~~Make protected stuff private by default. We already did this almost everywhere.~~
- ~~Remove since tags from private stuff.~~
- Make `@param` tags as specific as possible.
- Avoid full qualified class names.
- Remove a not needed `@var` tag.
- Fix my name.
- Update license tags according to SPDX version 3.
- Start using the PHPUnit's forward-compatible TestCase alias.
- Drop the custom PHPCS rule set, and use the upstream Wikibase rule set instead.

### CODE REVIEW

File: src/DimensionValidator.php [hunk 1]
```diff
@@ -15,57 +15,43 @@
 class DimensionValidator extends ValueValidatorObject {
 
 	/**
-	 * @since 0.1
-	 *
-	 * @var boolean
+	 * @var bool
 	 */
-	protected $allowAuto = false;
+	private $allowAuto = false;
```
Comment (JeroenDeDauw) [inline src/DimensionValidator.php]: this is a breaking change, cannot be done before next breaking rel, and might not make sense in there

Wikibase is not the only user of this lib btw

**Comment by JeroenDeDauw** [comment]
-2 as per inline comment

**Comment by thiemowmde** [comment]
Do you prefer this to be split in a separate patch, or not done at all?

**Comment by JeroenDeDauw** [comment]
The cost benefit of doing this seems very dubious. You need a breaking rel which is certainly not justified by just that change and need to verify such a change is fine for all users

**Comment by thiemowmde** [comment]
I reverted all the protected/private and since stuff for now.

**Comment by JeroenDeDauw** [comment]
+0

**Comment by mariushoch** [comment]
Needs rebase.

**Comment by thiemowmde** [comment]
Rebased and updated ([see above](https://github.com/DataValues/Validators/pull/10#issue-54668860)).

**Pull request merged**